### ISSUE TITLE
UI Builds up Memory Usage when left open (specifically on Gantt Execution chart)

### ISSUE THREAD

**Issue by japerry911** [kestra-io/kestra#5534]
### Describe the issue

- UI Builds up Memory Usage when left open (specifically on Gantt Execution chart)
- Leave UI open on running Execution on Gantt Chart, and it starts to leak memory I think

### Environment

- Kestra Version: 0.19.2
- Operating System (OS/Docker/Kubernetes): Kestra Cloud
- Java Version (if you don't run kestra in Docker): n/a

**Comment by code-grey** [comment]
Can I work on this? I'm learning to contribute (A lil nudge on where to get started will really be appreciated :)

**Comment by Skraye** [comment]
Hi @japerry911, is the memory usage going up only with a running flow ? so the gantt chart is animated ?

Hello @code-grey ! We don't have any details for now, you can look at the `Gantt.vue` component, as it seems the one generating the issue

To see it, you simply need to execute a Flow, and in the execution page there will be a tab with this gantt

**Comment by code-grey** [comment]
Thanks, I'll try and look into it :)

**Comment by japerry911** [comment]
@Skraye , correct, Gantt chart is animated/working, and the memory grows as I sit on gantt page.

**Comment by MilosPaunovic** [comment]
@japerry911 Not sure we can do much about it, but we'll definitely look into it. @code-grey you are welcome to take the first look and try to decrease memory usage.

**Comment by japerry911** [comment]
@MilosPaunovic sounds good, I'll try to share details if I can gather any, it's not terrible now, just noticed it this week since I've had a few longer flows running on second screen.

Thank you 🙏

**Comment by Skraye** [comment]
As I'm thinking of it, I wonder if the issue is not that, every second we receive an event from SSE, which is the current state of the execution
So every second, we may store the new state of the execution, without purging the previous, or maybe the SSE itself store every event and would need to be purged
Its a lead to follow I think

**Comment by MilosPaunovic** [comment]
If that's the case (and could easily be) then it can me amended and improved. Thanks for the guides, @Skraye! 🚀

**Comment by anna-geller** [comment]
@code-grey since it's been a month, I assume you no longer working on this :)

**Comment by code-grey** [comment]
> @code-grey since it's been a month, I assume you no longer working on this :)
> 
> 

I'm extremely sorry I wasn't able to continue my work.. semester exams are near and had to shift my focus into them.. I am sorry for not informing beforehand :)

**Comment by anna-geller** [comment]
@japerry911 is it still an issue in the current version? can you perhaps provide a flow or a screenshare on how to best reproduce it?

@Skraye @MilosPaunovic were you able to reproduce the issue? for now, assigning you @MilosPaunovic to have a look and to try to reproduce it

**Comment by japerry911** [comment]
Hello @anna-geller 

Feel free to close this. It has improved a lot in recent versions and I do not think it's issue currently 😃
